src/utils/get-ticket-no.ts

```diff
@@ -0,0 +1,26 @@
+export const IS_JIRA =
+  window.location.hostname.includes('atlassian.net') ||
+  window.location.pathname.startsWith('/browse/');
+
+export const IS_AZURE = window.location.hostname.includes('dev.azure.com');
+
+export const getTicketNo = () => {
+  const urlParams = new URLSearchParams(window.location.search);
```

Comment:
we can move these two lines to the else potion of the logic below since we don't need these for JIRA